### PR TITLE
readme: update for required direct ingress flags

### PR DESCRIPTION
**Pull request by sjenning**

### CODE REVIEW

File: cmd/cluster/create.go [hunk 1]
```diff
@@ -92,6 +92,7 @@ func NewCreateCommand() *cobra.Command {
 
 	cmd.MarkFlagRequired("pull-secret")
 	cmd.MarkFlagRequired("aws-creds")
+	cmd.MarkFlagRequired("base-domain")
```
Comment (csrwng) [inline cmd/cluster/create.go:95]: If we call `create infra` first and just pass the infra.json to `create cluster` is the `base-domain` still required?

File: README.md [hunk 2]
```diff
@@ -76,7 +79,8 @@ To delete the cluster and the infrastructure created earlier, run:
 hypershift destroy cluster \
   --aws-creds ~/.aws/credentials \
   --namespace clusters \
-  --name example
+  --name example \
+  --base-domain hypershift.example.com
```
Comment (ironcladlou) [inline README.md:83]: Why do we need to re-specify this on destroy? Seems like a UX regression to me

Comment (sjenning) [reply README.md:83]: If the HC is deleted independent the infra, then we have no way to look up the `base-domain` of a cluster of a particular name right?

Comment (ironcladlou) [reply README.md:83]: I agree, but isn't the typical case with this flow that we do in fact have the HostedCluster? In which case, we could use the discovered value by default and use the flag only if the HostedCluster isn't available.

**Pull request merged**